### DISCUSSION TITLE
Bài 6

### DISCUSSION THREAD

**Discussion by phucthinhb3** [phucthinhb3/Lop-11A03-Nhom-07#5] [General]
Phân chia công việc nha mấy bro
-Minh Khôi: 2 câu đầu
-Tuấn Anh: 2 câu kế, điền khiếm khuyết
-Phúc Thịnh, Tín và Phong làm các câu còn lại nhé
Làm nhớ đúng hạn nha mấy man

**Comment by tuananh-b3-03** [comment]
okk

**Reply by trinhtienphong** [reply]
okk

**Comment by Minh-Khoi16** [comment]
好的

**Comment by trinhtienphong** [comment]
okkk

**Reply by tintruongvinh** [reply]
ok hiểu rồi
